replication_impl.go

```diff
@@ -139,13 +141,15 @@ func (b batchMetadata) Extend(ctx context.Context, ttl time.Duration) error {
 // Delete an existing dump batch
 func (b *batchMetadata) Delete(ctx context.Context) error {
 	if !atomic.CompareAndSwapInt32(&b.closed, 0, 1) {
-		return WithStack(errors.New("Batch already closed"))
+		return WithStack(ErrBatchClosed)
 	}
 
 	req, err := b.cl.conn.NewRequest("DELETE", path.Join("_db", b.database, "_api/replication/batch", b.ID))
 	if err != nil {
 		return WithStack(err)
 	}
+
+	req = req.SetQuery("serverId", strconv.FormatInt(b.serverID, 10))
```

Comment:
Q: Is this a requirement for all arango versions?

Reply:
I have reviewed documentation and "serverId" is not used from at least version 3.3. I copied that from arangosync because arangosync uses that but it is not necessary so I discard that change